src/Main.js

```diff
@@ -74,12 +74,31 @@ WebhookHandler.on('push', HandlerWrap((ev) => {
     }
 }));
 
+// prUrl format, e.g., https://api.github.com/repos/github/hello-world/pulls/1
+// whether the PR belongs to Squid Project
+function isPrValid(prUrl) {
+    const basePath = Config.baseUrl() + '/repos/';
+    assert(prUrl.startsWith(basePath));
+    const arr = prUrl.substring(basePath.length).split('/');
+    assert(arr.length === 4);
+    return arr[0] === Config.owner();
```

Comment:
Please check the repository name as well.

Reply:
Ok.